subtitle_video.py

```diff
@@ -153,9 +153,9 @@ def transcribe_api(openai_client, audio_path):
         "text": transcription.text,
         "segments": [
             {
-                "start": segment["start"],
-                "end": segment["end"],
-                "text": segment["text"]
+                "start": segment.start,
```

Comment:
Did you test these changes? It breaks the code for me. I would revert these changes back to what they were (using the ["start"], ["end:] and ["text"] dictionary indexing